### ISSUE TITLE
ipywidgets: hs.load()

### ISSUE THREAD

**Issue by magnunor** [hyperspy/hyperspy#1642]
Currently, when trying to use `hs.load()` with the ipywidgets gui, you get the `NotImplementedError: There is no user interface registered for this feature.Try installing hyperspy_gui_traitsui.`

There should maybe be a specialized error for `hs.load()`, since it is such a commonly used function. Something like:
`NotImplementedError: There is no user interface registered for load(), use load("path/to/filename") instead, or use another user interface by installing hyperspy_gui_traitsui.`

**Comment by francisco-dlp** [comment]
I agree that the message is not as good as it could be but, on the other hand, it is automatically generated by a function that knows nothing about ``load``. We could "hack" the message but, why doing it for ``load`` and not for others? On the other hand, if we go down this path, the docstring will be harder to keep up-to-date.

**Comment by thomasaarholt** [comment]
We should be able to put together at least a temporary solution for load:
https://gist.github.com/DrDub/6efba6e522302e43d055

This one puts together a very rudimentary layout inline in the notebook of files and folders in the current working directory and "onwards". It's got some bugs (needs a os.path.join instead of "/", but should be easy to improve on.
